legacy/routes/multiapi.js

```diff
@@ -99,6 +100,11 @@ var multiapi = function(optionalCoverage) {
             utils.send400(res, next, "The api version of testFive is not supported: " + req.query['api-version']);
         }
     });
+
+    router.post('/lroAndPaging', function (req, res, next) {
```

Comment:
i wanted to keep this place extra simple, i was just trying to focus on the generation of the code. Let me know if I should make this more complex